docs/commands/lmmreg.md

```diff
@@ -0,0 +1,173 @@
+`global.lmmreg.maxLogLkhd` | Double | (restricted) maximum log likelihood corresponding to the fit delta
+
+These global annotations are also added to `hail.log`, with the ranked evals and $\delta$ grid with values in `tsv` tabular form.  Use `grep 'lmmreg:' hail.log` to find the lines just above each table.
+
+If Step 5 is performed, `lmmreg` also adds nine variant annotations.
+
+Annotation | Type | Value
+---|---|---
+`va.lmmreg.beta` | Double | fit genotype coefficient, $\hat\beta_0$
+`va.lmmreg.sigmaG2` | Double | fit coefficient of genetic variance component, $\hat{\sigma}_g^2$
+`va.lmmreg.chi2` | Double | $\chi^2$ statistic of the likelihood ratio test
+`va.lmmreg.pval` | Double | $p$-value
+`va.lmmreg.maf` | Double | minor allele frequency for included samples
+`va.lmmreg.nHomRef` | Int | count of HomRef genotypes for included samples
+`va.lmmreg.nHet` | Int | count of Het genotypes among for samples
+`va.lmmreg.nHomVar` | Int | count of HomVar genotypes for included samples
+`va.lmmreg.nMissing` | Int | count of missing genotypes for included samples
+
+Unlike the `filtervariants` command, the `-k` (`--kinshipfilter`) and `-a` (`--assocfilter`) filters do *not* remove variants from the underlying variant dataset.  Rather, variants for which `va.inAssoc` is false will have missing values for these annotations, as will those variants that don't vary across the included samples (e.g., all genotypes are HomRef). If both the kinship variants and association variants reside is a subset of the genome, consider running `filtervariants` before `lmmreg`.  You can also `filtervariants` after running `lmmreg` but before running `exportvariants` to avoid outputting variants with missing annotations.  The simplest way to export all resulting annotations is:
+
+```
+  showglobals \
+    -o lmmreg.json \
+  exportvariants \
+    -c 'variant = v, va.lmmreg.*' \
+    -o lmmreg.tsv.bgz
+```
+
+Similarly `lmmreg` does *not* remove samples from the underlying variant dataset.
+
+#### Performance
+
+This initial version of `lmmreg` scales efficiently to tens of thousands samples and an unbounded number of variants, making it particularly well-suited to modern sequencing studies. For example, starting from a VDS of the 1000 Genomes Project (consisting of 2535 whole genomes), `lmmreg` computes a kinship matrix based on 100k common variants, fits coefficients and variance components in the sample-covariates-only model, runs a linear-mixed-model likelihood ratio test for all 15 million high-quality non-rare variants, and exports the results in under 4 minutes. Here we used 42 preemptible workers (~680 cores) on 2k partitions at a cost of about 50 cents on Google cloud (see [Using Hail on the Google Cloud Platform](http://discuss.hail.is/t/using-hail-on-the-google-cloud-platform/80)). The first analysts to apply `lmmreg` in research have found the method takes under an hour on 8185 samples and 25 million variants, with 262k variants used to compute kinship.
```

Comment:
I did a quick calculator for 42 n1-standard-16 workers and 1 hour looks like 34 USD. Where did you get the 50 cents number from?